targets/UnrealMarketplacePlugin/source/PlayFab/PlayFab.uplugin.ejs

```diff
@@ -2,7 +2,7 @@
     "FileVersion": 3,
     "FriendlyName": "PlayFab Marketplace Plugin",
     "Version": 0,
-    "EngineVersion": "<%- ueTargetVersion %>",
+    "EngineVersion": "<%- ueTargetVersion %>.0",
```

Comment:
I would rather have two variables that come from make.js:
ueTargetVersionShort and ueTargetVersionLong
Because that final micro version number is not always zero.

Reply:
Actually I'll probably do a bit more than that. I need to use the Major version to generate different build command line in the batch file. So I'll reuse the same format than in UE source: version major, minor and patch, eventually adding TargetVersion Short and Long for ease of use too.